### ISSUE TITLE
Adjust how drakes are handled

### ISSUE THREAD

**Issue by bulldogc** [chummer5a/chummer5a#5040]
Applying drake powers to a base character seems counterproductive, especially with stat adjustments and natural weapons ECT which only apply to the drake form not the metahuman form. A better way to manage drakes might be to treat them like shifters and give them a "drake form" selection which changes their stats and powers ect.

**Comment by Maguillage** [comment]
The section on "creating drake characters" doesn't agree with the "drake" quality's listed benefits, so mostly we just have to blame poor editing.
My take on it is that you always gain the relevant attribute increases, but when actually in drake form, it also applies to your metatype maximum.
So a 4 str human eastern drake has effectively 6 str, but no further benefit in drake form, while a 6 str eastern drake human has 6 str as expected, and can jump to 8 str in drake form.

**Comment by DelnarErsike** [comment]
Yeah, this is one of those spots where RAW is just flat out dumb, but because Chummer5a is meant to represent RAW wherever possible, the existing way of handling drakes has to stay. If someone wishes to add a homebrew system that makes them work the way you describe, I wouldn't be against that. So I'm tagging this as a feature request.
